students/JerryH/Lesson07/test_html_render.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+import html_render as hr
+from io import StringIO
+
+
+class TestHtmlRender(unittest.TestCase):
+
+    def test_element(self):
+        test_element = hr.Element("test content")
+        self.assertEqual(test_element.content, ["test content"])
+        test_element.tag = "p"
+        self.assertEqual(test_element.tag, 'p')
+        test_element.tag = 'body'
+        self.assertEqual(test_element.tag, 'body')
```

Comment:
tests in lines 11 to 14 - I would question the need for tests like this, this is essentially testing that python can do it's job.